persist/file_body_persister.go

```diff
@@ -0,0 +1,94 @@
+package persist
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+
+	"github.com/jmartin82/mmock/definition"
+	"github.com/jmartin82/mmock/parse"
+)
+
+//FileBodyPersister persists body in file
+type FileBodyPersister struct {
+	PersistPath string
+	Parser      parse.ResponseParser
+}
+
+//Persist the body of the response to fiel if needed
+func (fbp FileBodyPersister) Persist(per *definition.Persist, req *definition.Request, res *definition.Response) {
+	if per.Name == "" {
+		return
+	}
+
+	per.Name = fbp.Parser.ReplaceVars(req, per.Name)
+
+	filePath := path.Join(fbp.PersistPath, per.Name)
+	fileDir := path.Dir(filePath)
+
+	if per.Delete {
+		os.Remove(filePath)
+	} else {
+		fileContent := []byte(res.Body)
+		err := os.MkdirAll(fileDir, 0644)
+		if checkForError(err, res) == nil {
+			err = ioutil.WriteFile(filePath, fileContent, 0644)
+			checkForError(err, res)
+		}
+	}
+}
+
+func (fbp FileBodyPersister) getFilePath(fileName string, req *definition.Request) (filePath string, fileDir string) {
+	fileName = fbp.Parser.ReplaceVars(req, fileName)
+
+	filePath = path.Join(fbp.PersistPath, fileName)
+	fileDir = path.Dir(filePath)
```

Comment:
I'd like add one check here to verify if the resulting paths is still inside in the PersistPath to avoid the possibility to extract information from other files.